src/server.rs

```diff
@@ -718,24 +704,25 @@ impl ServerStats {
     fn to_cache_statistics(&self) -> Vec<CacheStatistic> {
```

Comment:
We could probably do better here now just by defining the `CacheStats` type to use the `CacheStat` type for its fields directly, but we can sort that out in a followup.